dd-trace-api/src/main/java/datadog/trace/api/ConfigDefaults.java

```diff
@@ -101,6 +102,16 @@ public final class ConfigDefaults {
   static final boolean DEFAULT_JMX_FETCH_MULTIPLE_RUNTIME_SERVICES_ENABLED = false;
   static final int DEFAULT_JMX_FETCH_MULTIPLE_RUNTIME_SERVICES_LIMIT = 10;
 
+  static final boolean DEFAULT_METRICS_OTEL_ENABLED = false;
+  static final OtelConfig.Exporter DEFAULT_OTEL_METRICS_EXPORTER = OtelConfig.Exporter.OTLP;
+  // WARNING: This defies the OpenTelemetry specification’s default value of 60000 (60s)
+  static final Integer DEFAULT_OTEL_METRIC_EXPORT_INTERVAL = 10000;
+  // WARNING: This defies the OpenTelemetry specification’s default value of 30000 (30s)
+  static final Integer DEFAULT_OTEL_METRIC_EXPORT_TIMEOUT = 7500;
+  static final OtelConfig.Protocol DEFAULT_OTEL_EXPORTER_OTLP_METRICS_PROTOCOL =
+      OtelConfig.Protocol.GRPC;
```

Comment:
```suggestion
  static final Protocol DEFAULT_OTEL_EXPORTER_OTLP_METRICS_PROTOCOL =
      Protocol.GRPC;
```

With the edited imports above, we can simplify this line.